class.jetpack.php

```diff
@@ -613,6 +613,11 @@ private function __construct() {
 		// A filter to control all just in time messages
 		add_filter( 'jetpack_just_in_time_msgs', '__return_true', 9 );
 
+		// If enabled, point edit post and page links to Calypso instead of WP-Admin.
+		if ( get_option( 'jetpack_edit_links_calypso_redirect' ) ) {
```

Comment:
@rralian instead of separate option here, would it maybe be more suitable to add a custom filter that you can enable from `wpcomsh`? I imagine that would be easier for you than setting this option for all AT sites (or making sure that it's set during transfer).

Reply:
Are we going to have an UI option for this? If not, then I like a filter better. If that's the case, looks like we can set this up directly in `wpsh` and might not need it in Jetpack at all.

I'm not sure the roadmap long-term, though.

Reply:
I believe the idea is to eventually have a UI (not this initial PR) that would allow Jetpack users to choose between keeping `wp-admin` as the destination, or switching to using Calypso; in this case, we'd need to stick with the option. Although, this brings up the question: would we want it to be a per-site option, or a user preference? I'm thinking the latter would be a better fit.

Reply:
@rralian wanted to avoid adding this to `wpcomsh` if possible. Based on that, I guess the long term goal is to have this option available for Jetpack users in general - so might be better to stick with option after all.

Reply:
Agreed, we want to first address this for Atomic as a mandatory setting, and are trying to get this into the Jetpack release so we can do so quickly. We would also like to provide this as an option in Jetpack, but we can take our time with providing a UI for the option in the next Jetpack release.